chat/src/main/java/com/github/twitch4j/chat/events/channel/ChannelRemovedPostJoinFailureEvent.java

```diff
@@ -0,0 +1,49 @@
+package com.github.twitch4j.chat.events.channel;
+
+import lombok.Value;
+
+/**
+ * A chat instance failed to join (or stay joined to) a channel AND removeChannelOnJoinFailure was enabled,
+ * resulting in removal from the instance's current channels.
+ */
+@Value
+public class ChannelRemovedPostJoinFailureEvent {
```

Comment:
is this event name too verbose

Reply:
What do you think about naming it `ChannelJoinFailureEvent`?